src/coverlet.msbuild.tasks/MSBuildLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ILogger = Coverlet.Core.ILogger;
+
+namespace Coverlet.MSbuild.Tasks
+{
+    class MSBuildLogger : ILogger
+    {
+        private readonly TaskLoggingHelper _log;
+
+        public MSBuildLogger(TaskLoggingHelper _log)
+        {
+            this._log = _log;
+        }
+
+        public void LogSuccess(string message)
```

Comment:
@tonerdo I don't see any usage of this method https://github.com/tonerdo/coverlet/search?q=LogSuccess&unscoped_q=LogSuccess could be removed?